src/api/user/jest.config.js

```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const baseConfig = require('../../../jest.config.base');
+require('dotenv').config({
+  path: path.join(__dirname, '../env.development'),
+});
+
+// FIRESTORE_EMULATOR_HOST requires an address to be set in an env file
+// in order for firestore to be ran locally
+process.env = { ...process.env, FIRESTORE_EMULATOR_HOST: 'localhost:8088' };
```

Comment:
This isn't needed, since you put it in `env.development` already.

Reply:
This is needed for Jest to be able to run the tests

Reply:
If you pull in the env, you don't need to redefine it though, since you define it in the env.